conf/routes

```diff
@@ -6,5 +6,5 @@ GET        /alive                                             controllers.Health
 GET        /install/beta                                      controllers.InstallController.install(beta: Boolean = true, rcupdate: Option[Boolean])
 GET        /install/stable                                    controllers.InstallController.install(beta: Boolean = false, rcupdate: Option[Boolean])
 GET        /selfupdate/beta                                   controllers.SelfUpdateController.selfUpdate(beta: Boolean = true)
-GET        /selfupdate/stable                                 controllers.SelfUpdateController.selfUpdate(beta: Boolean = false)
+GET        /selfupdate/stable                                 controllers.SelfUpdateController.selfUpdate(beta: Boolean ?= false)
```

Comment:
Is the question mark here correct?

Reply:
Yes, that represents an [optional query parameter with default value](https://www.playframework.com/documentation/2.8.x/ScalaRouting#Parameters-with-default-values). This is the vital change for backward compatibility to work.

Reply:
Very nice!